files/pigeon-js/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "pigeon-js",
+  "repo": "labsvisual/pigeon-js",
+  "files": {
+    "basePath": "dist/"
```

Comment:
Only `pigeon.min.js` is needed in production. Please add an include parameter for that file